pyrfr/MANIFEST.in

```diff
@@ -1,3 +1,4 @@
 recursive-include include *
 recursive-include pyrfr *.i
-include LICENSE_cereal-1.2.2
\ No newline at end of file
+include LICENSE_cereal-1.2.2
+include pyproject.toml
```

Comment:
Could you please include the regular license, too, that I just included via a PR?